scripts/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) 2019
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+# Collects EdgeX Go binaries (must be previously built)
+# and packages them into a dir alongside with necessary config files.
+# This facililtets transfer of binaries to otehr machine
+# (for example copy of cross-compiled binaris to ARM gateway)
+
+DIR=$PWD
+CMD_DIR=../cmd
+SERVICES=(config-seed export-client core-metadata core-command support-logging \
+  support-notifications sys-mgmt-executor sys-mgmt-agent support-scheduler \
+  core-data export-distro)
+
+PKG_DIR=${1:-edgex}
+
+# Copy binaries and config
+mkdir -p $PKG_DIR
+for i in "${SERVICES[@]}"; do
+  mkdir -p $PKG_DIR/$i
+  cp $CMD_DIR/$i/$i $PKG_DIR/$i
+  cp $CMD_DIR/$i/res/configuration.toml $PKG_DIR/$i 2>/dev/null || :
```

Comment:
won't this copy the configuration.toml files inside `$PKG_DIR/core-data/configuration.toml` instead of the standard `$PKG_DIR/core-data/res/configuration.toml`? I think this should probably be consistent with our other distribution layouts and go inside `res`

Reply:
Yes, my idea of the "package" was something simple that I can easily copy on a gateway - for example put it on a USB stick, or send it via `scp`.

I do not see the benefit of having `./res` here - I would rather see a config file right next to the binary then have to go one level more in the `res` directory.

Moreover, these `res` directories today contain `docker` dir, which is completely not needed when we have these native binaries - so we would be creating `res` directory only to put one `config.toml` in it. I find this unnecessary complexity and overengineering.

Reply:
FWIW, I agree with you and I don't understand why we have all these `res` directories, perhaps @tsconn23 can comment on why that is.
I do still think that this packaging script should follow the pattern of edgex-go itself, so perhaps the thing to do here is to also remove all of the `res` directories in the edgex-go.

Lastly, we follow the same pattern in the snap, simply because that's what edgex-go does and so we try to follow that.

Reply:
The `./res` directories have been with the project longer than I have. Their purpose has always been to hold configuration information for the default / docker profiles as well as any legacy java *.properties files (see config-seed/res/properties). If you would like to change the way configuration files are organized and loaded, file an issue.

Reply:
I will open an issue once I finish my tests, but basically idea is to pick up only non-docker config in some easy way when creating a binary package.